models/gamesM.js

```diff
@@ -20,3 +20,25 @@ exports.selectReviews = () => {
       return result.rows;
     });
 };
+
+exports.selectReviewById = (review_id) => {
+  return pool
+    .query(
+      `SELECT  reviews.review_id, title, 
+      review_body, designer, review_img_url, 
+      reviews.votes, reviews.category, owner, 
+      created_at FROM reviews
+        WHERE reviews.review_id=$1
+        GROUP BY reviews.review_id;`,
```

Comment:
An unnecessary GROUP BY here, it wont cause you problems as of yet but feels a bit pointless to group when we arent using any aggregate functions